challenge/src/components/Pages/FunFacts.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+ import GIF from "../Gif"
+import Table from '../Table'
+
+
+
+const FunFacts = (props) => {
+   
+    return (
+        <div>
+            
+            <h1 style={{ fontSize: 100, color: "#FFF5B8" }}>
+              <center>Fun Facts</center>
+            </h1>
+
+
```

Comment:
I don't mean to be nit-picky but this kind of seems like a lot of whitespace/newlines

Reply:
> I don't mean to be nit-picky but this kind of seems like a lot of whitespace/newlines

Sorry, I didn't realize formatting was messed up before, will fix it!